gateway-service/src/test/resources/gateway-messages.yml

```diff
@@ -231,3 +231,17 @@ messages:
       reason: "An error occurred in the SAF Auth Service. Review the reason in the error message."
       action: "Supply a valid user and application name, and check that corresponding permissions have been set up."
 
+  # IDT messages (150 - 159)
+    - key: org.zowe.apiml.security.idt.failed
+      number: ZWEAG150
+      type: ERROR
+      text: "SAF IDT generation failed. Reason: %s"
+      reason: "An error occurred in the SAF Verify Service. Review the reason in the error message."
```

Comment:
Same as above

Reply:
Based on the discussion with Andrew he agreed that the proposed wording is better.